BUILD.bazel

```diff
@@ -16,7 +16,7 @@ gazelle(
 filegroup(
     name = "all-images",
     srcs = [
-        "//clusterctl:clusterctl-image",
+        "//cmd/clusterctl:clusterctl-image",
         "//cmd/clusterawsadm:clusterawsadm-image",
```

Comment:
Am I misreading this? Is this creating a container image for clusterctl and clusterawsadm? @liztio